frontend/src/profile/Profile.tsx

```diff
@@ -301,15 +299,10 @@ const ProfileAdvisor = (props: any) => {
   const save = () => {
     setEdit(false);
     batch(() => {
-      dispatch(setStudentMajorAction(major || ""));
+      dispatch(setStudentMajorAction(major));
       dispatch(setStudentCatalogYearAction(catalogYear));
       dispatch(setStudentConcentrationAction(concentration));
-      if (coopCycle !== "None Selected") {
-        dispatch(setStudentCoopCycleAction(""));
-      } else {
-        dispatch(setStudentCoopCycleAction(coopCycle || ""));
-      }
-
+      dispatch(setStudentCoopCycleAction(coopCycle));
```

Comment:
I've tested that the dropdowns which display "None selected" dispatch `null` rather than "None selected" even with this change.